ADTTools/UploadModels/Program.cs

```diff
@@ -143,8 +144,8 @@ private static bool ParseModelJson(Dictionary<string, string> modelTexts)
             IReadOnlyDictionary<Dtmi, DTEntityInfo> entities = null;
             try
             {
-                var parser = new ModelParser();
-                entities = await parser.ParseAsync(modelTexts.Values);
+                var parser = new ModelParser(new ParsingOptions() { AllowUndefinedExtensions = true });

```

Comment:
This seems like a value that should be explicitly set by the user of ADTTools to avoid production breaks.